typescript/ecs/ecs-service-with-advanced-alb-config/index.ts

```diff
@@ -46,9 +46,9 @@ listener.addTargets('ECS', {
   targets: [service],
   // include health check (default is none)
   healthCheck: {
-    intervalSecs: 60,
+    interval: cdk.Duration.seconds(60),
```

Comment:
Last [commit](https://github.com/awslabs/aws-cdk/commit/fa29d3b) I have seen to `healthCheck` indicates that the original method is correct. We should keep that

Reply:
@NGL321 I just update to the latest version and `intervalSecs` does not exist. Was there anything else you are looking for?

Reply:
I will accept this one for now, and I'll investigate further (we may just need a documentation update).

Reply:
Awesome. I think this resolves this issue.